imsim/ccd.py

```diff
@@ -169,6 +169,7 @@ def parse(item, type, default):
         image.header['DAYOBS'] = dayobs
         image.header['SEQNUM'] = seqnum
         image.header['CONTRLLR'] = 'P', 'simulated data'
+        image.header['RUNNUM'] = parse('observationId', int, -999)
```

Comment:
The -999 is just a default value.  For opsim db input, `observationId` is provided from the `observations` table.  Similarly, in an instcat file, the `obshistid` entry provides this value.   It may be that the default value here is never used, but the `parse` interface requires that argument to be provided.